tests/python_client/testcases/test_search.py

```diff
@@ -6660,3 +6687,540 @@ def test_range_search_with_consistency_session(self, nq, dim, auto_id, _async):
+        expected: No exception
+        """
+        # insert data
+        collection_w = self.init_collection_general(prefix, partition_num=1)[0]
+        partition_w1, partition_w2 = collection_w.partitions
+        df = cf.gen_default_dataframe_data(200)
+        partition_w1.insert(df[:100])
+        partition_w2.insert(df[100:])
+        # load
+        collection_w.load()
+        # delete data
+        delete_ids = [i for i in range(50, 150)]
+        collection_w.delete(f"int64 in {delete_ids}")
+        # release
+        partition_w1.release()
+        # search
+        collection_w.search(vectors[:1], default_search_field, default_search_params,
+                            default_limit, default_search_exp, partition_names=[partition_w1.name],
+                            check_task=CheckTasks.err_res,
+                            check_items={ct.err_code: 1, ct.err_msg: 'not loaded'})
+        res = collection_w.search(vectors[:1], default_search_field, default_search_params,
+                                  100, default_search_exp, partition_names=[partition_w2.name])[0]
+        assert len(res[0]) == 50
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_load_partition_delete_release_collection(self):
+        """
+        target: test delete load collection release partition
+        method: 1. create a collection and 2 partitions
+                2. insert data
+                3. load one partition
+                4. delete half data in each partition
+                5. release the collection
+                6. search on the partition -> raise error
+        expected: No exception
+        """
+        # insert data
+        collection_w = self.init_collection_general(prefix, partition_num=1)[0]
+        partition_w1, partition_w2 = collection_w.partitions
+        df = cf.gen_default_dataframe_data(200)
+        partition_w1.insert(df[:100])
+        partition_w2.insert(df[100:])
+        # load
+        partition_w1.load()
+        # delete data
+        delete_ids = [i for i in range(50, 150)]
+        collection_w.delete(f"int64 in {delete_ids}")
+        # release
+        collection_w.release()
+        # search
+        collection_w.search(vectors[:1], default_search_field, default_search_params,
+                            default_limit, default_search_exp,
+                            check_task=CheckTasks.err_res,
+                            check_items={ct.err_code: 1, ct.err_msg: 'not loaded'})
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_load_partition_delete_drop_partition(self):
+        """
+        target: test load partition delete drop partition
+        method: 1. create a collection and 2 partitions
+                2. insert data
+                3. load one partition
+                4. delete half data in each partition
+                5. drop the non-loaded partition
+                6. search on the dropped partition -> len(res)==50
```

Comment:
search on the non-dropped partition?

Reply:
> search on the non-dropped partition?

Here should be search on the non-dropped partition. I'll correct this sentence.